src/provider/provider_level_zero.c

```diff
@@ -61,6 +61,14 @@ umf_result_t umfLevelZeroMemoryProviderParamsSetResidentDevices(
     return UMF_RESULT_ERROR_NOT_SUPPORTED;
 }
 
+umf_result_t umfLevelZeroMemoryProviderParamsSetFreePolicy(
```

Comment:
Could you please update the test in the `provider_level_zero_not_impl.cpp` file?

Reply:
Done.